open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/buffergraph/MainGraphMapper.java

```diff
@@ -4,72 +4,302 @@
+        Iterator<Vertex> schema = bufferG.V(originalVertex.id()).emit().repeat(bothE().inV().simplePath()).times(3).
+                or(hasLabel(RELATIONAL_DB_SCHEMA_TYPE),hasLabel(FILE_FOLDER));
+
+        //find a query for filefolder parent
+//        Vertex db = bufferTraversal.V(originalVertex.id()).until(or(hasLabel(DATABASE),hasLabel(FILE_FOLDER))).repeat(bothE().inV().simplePath()).next();
+        if(tableAsset.hasNext()){
+            newVertex.property(PROPERTY_NAME_TABLE_DISPLAY_NAME,tableAsset.next().property("vepropdisplayName").value());
+        }
+
+        if(schema.hasNext()){
+            newVertex.property(PROPERTY_NAME_SCHEMA_DISPLAY_NAME,schema.next().property("vepropdisplayName").value());
+        }
+
+//        if(db != null){
+//            mainTraversal.V(newVertex.id()).property(PROPERTY_NAME_DATABASE_DISPLAY_NAME,db.property(PROPERTY_KEY_ENTITY_NAME).value());
+//        }
+
+        getGlossaryTerm(mainG,bufferG,newVertex);
+
+    }
+
+    /**
+     * Add glossary term related to the asset
+     *
+     * @param asset  - Asset to find glossary term
+     * */
+    private void getGlossaryTerm(GraphTraversalSource mainG,GraphTraversalSource bufferG,Vertex asset){
+
+        Iterator<Vertex> glossaryTermBuffer = bufferG.V()
+                                                     .has(PROPERTY_KEY_ENTITY_GUID,asset.property(PROPERTY_KEY_ENTITY_GUID).value().toString())
+                                                     .bothE("SemanticAssignment")
+                                                     .outV();
+
+        if(glossaryTermBuffer.hasNext()) {
+            String guidGlossary = glossaryTermBuffer.next().property(PROPERTY_KEY_ENTITY_GUID).value().toString();
+
+           Vertex glossaryMain =   mainG.V().has(PROPERTY_KEY_ENTITY_GUID, guidGlossary).
+                    fold().
+                    coalesce(unfold(),
+                            addV(NODE_LABEL_GLOSSARYTERM)).property(PROPERTY_KEY_ENTITY_NODE_ID,guidGlossary).next();
+            mainG.V(asset.id()).as("v").
+                        V(glossaryMain.id()).
+                        coalesce(__.outE(EDGE_LABEL_SEMANTIC).where(inV().as("v")),
+                                addE(EDGE_LABEL_SEMANTIC).from("v")).next();
+        }
+
+        //TODO copy glossaryterm
+    }
+
+    /**
+     * Add the Process and the relationships to the Process to the graph
+     *
+     * @param columnInVertex  - Initial column tha is processed
+     * @param columnOutVertex - Column coming out of the process
+     * @param process         - Process to add relationships
+     *
+     * */
+    private void addProcess(Vertex columnInVertex,Vertex columnOutVertex,Vertex process){
+
+        GraphTraversalSource mainG = mainGraph.traversal();
 
         final String processGuid = process.value(PROPERTY_KEY_ENTITY_GUID);
-        final String processName = process.value(PROPERTY_NAME_NAME);
+        final String processName = process.value("vepropdisplayName");
```

Comment:
Hardcoded string, "vepropdisplayName" can be found as constant PROPERTY_KEY_ALTERNATIVE_DISPLAY_NAME in GraphConstants. Maybe it should be "vepropDisplayName"?